src/QMCWaveFunctions/Fermion/BackflowTransformation.h

```diff
@@ -740,7 +739,6 @@ class BackflowTransformation //: public OrbitalSetTraits<QMCTraits::ValueType>
       dr[1] = 0.05;
       dr[2] = -0.3;
       P.makeMove(iat, dr);
-      const auto& myTable = P.getDistTable(myTableIndex_);
```

Comment:
Same issue with AoS build

Reply:
should be fixed now and ready for review